src/hammer-tech/specialcells.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+#  stackup.py
+#  Data structures to represent technology's special cells.
+#
+#  See LICENSE for licence details.
+
+from enum import Enum
+from typing import List, NamedTuple, Tuple, Dict, Optional
+from hammer_utils import reverse_dict
+from decimal import Decimal
+
+class CellType(Enum):
+    TieCell = 1
+    EndCap = 2
+    IOFiller = 3
+    StdFiller = 4
+
+    @classmethod
+    def __mapping(cls) -> Dict[str, "CellType"]:
```

Comment:
It's a classmethod since you can call it with a reference to the class, whereas IIRC static methods have no such capability.

Reply:
Well, static methods have no sense of state, whereas class methods can access a global state that is associated with the class rather than its objects. This is stateless which makes me think it should be a static method.